README.md

```diff
@@ -7,13 +7,120 @@ api-filter
 
 Parser/builder for filters from api query parameters.
 
+It is just a parser/builder for filters, it is not a place for business logic so it should be wrapped by your class, if you want to be more strict about filters.
+Same if you want different settings per entity/table, it should be done by specific wrapper around this library.
+
+
 ## Installation
 ```bash
 composer require lmc/api-filter
 ```
 
 
+## Filters
+
+### EQ (=)
```

Comment:
Would be nice for newcommers exaplin this cryptic shortcuts  (LT, LTE, EQ...).

Rule of thumb to make README readable is to never use shortcuts in readme and examples.